src/commands/server/start.ts

```diff
@@ -186,6 +186,20 @@ export default class Start extends Command {
 
   checkPlatformCompatibility(flags: any) {
     // matrix checks
+    if (flags.installer === 'operator' && flags['che-operator-cr-yaml']) {
+      this.warn('--che-operator-cr-yaml is used. The following flags will be ignored:')
```

Comment:
Does it make sense not to show this message if user did not specify any of the listed parameters?